### PR TITLE
Added support of passing options for berkshelf via knife.rb

### PR DESCRIPTION
**Pull request by iavael**

### CODE REVIEW

**Comment by tmatilai** [comment]
@iavael the tests use Berkshelf 3, but knife-solo itself is also compatible with v2 (see #268).

**Comment by iavael** [comment]
> the tests use Berkshelf 3, but knife-solo itself is also compatible with v2

Then why
`s.add_development_dependency 'berkshelf', '>= 3.0.0.beta.2'`
is written in gemspec?
Anyway berkshelf 3 would be installed.

File: lib/knife-solo/berkshelf.rb [hunk 1]
```diff
@@ -19,7 +19,12 @@ def install!
       path = berkshelf_path
       ui.msg "Installing Berkshelf cookbooks to '#{path}'..."
 
-      berksfile = ::Berkshelf::Berksfile.from_file('Berksfile')
+      if defined?(::Berkshelf) && Gem::Version.new(::Berkshelf::VERSION) >= Gem::Version.new("3.0.0")
```
Comment (matschaffer) [inline lib/knife-solo/berkshelf.rb]: Didn't realize you'd added a compatibility check. Thanks!

What's the reason for the `defined?(::Berkshelf)` check here? Seems like if it failed the original code as well as line 26 here will fail as well.

Comment (iavael) [inline lib/knife-solo/berkshelf.rb]: Well, you are right, there is no reason. I removed it.

**Comment by matschaffer** [comment]
Thanks!

**Pull request merged**